lib/Page.js

```diff
@@ -1386,9 +1379,6 @@ class FileChooser {
   async cancel() {
```

Comment:
I suppose we left this method "async" to be consistent with accept() and be backward-compatible?

Reply:
Indeed, [`FileChooser.cancel`](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#filechoosercancel) is part of the public API, and should continue to return a promise (or at least a thenable) since code might depend on it.